commands/kubernetes.go

```diff
@@ -613,6 +614,20 @@ func (s *KubernetesCommandService) RunKubernetesClusterCreate(defaultNodeSize st
 			s.tryUpdateKubeconfig(kube, cluster.ID, clusterName, setCurrentContext)
 		}
 
+		oneClickApps, err := c.Doit.GetStringSlice(c.NS, doctl.ArgOneClicks) 
+		if err != nil {
+			return err
+		}
+		if len(oneClickApps) > 0 {
```

Comment:
What happens if `wait` is false here? e.g. `doctl kubernetes cluster create --wait=false new-cluster-01 --1-clicks loki`
Is it the install job queued or does the cluster need to be completely up first?

Reply:
The install job is queued, it waits for the cluster to be up and then installs the application.

Reply:
:ok_hand:  Perfect!